### PR TITLE
Search Customer Api Added

### PR DESCRIPTION
**Pull request by meeta-tgs**
search customer by id , name and both logic added
Exception handler also added

### CODE REVIEW

File: logs/customer-service/customer-service.log [hunk 1]
```diff
@@ -0,0 +1,705 @@
...
+2020-02-05 13:54:34.245 DEBUG 27156 --- [restartedMain] o.s.web.context.ContextLoader            : Published root WebApplicationContext as ServletContext attribute with name [org.springframework.web.context.WebApplicationContext.ROOT]
+2020-02-05 13:54:34.246  INFO 27156 --- [restartedMain] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 4348 ms
+2020-02-05 13:54:35.067  INFO 27156 --- [restartedMain] o.hibernate.jpa.internal.util.LogHelper  : HHH000204: Processing PersistenceUnitInfo [name: default]
+2020-02-05 13:54:35.240  INFO 27156 --- [restartedMain] org.hibernate.Version                    : HHH000412: Hibernate Core {5.4.10.Final}
+2020-02-05 13:54:35.534  INFO 27156 --- [restartedMain] o.hibernate.annotations.common.Version   : HCANN000001: Hibernate Commons Annotations {5.1.0.Final}
+2020-02-05 13:54:35.757  INFO 27156 --- [restartedMain] com.zaxxer.hikari.HikariDataSource       : HikariPool-1 - Starting...
+2020-02-05 13:54:37.604 ERROR 27156 --- [restartedMain] com.zaxxer.hikari.pool.HikariPool        : HikariPool-1 - Exception during pool initialization.
+
+java.sql.SQLSyntaxErrorException: Unknown database 'digipoc'
```
Comment (laburli) [inline logs/customer-service/customer-service.log]: why this error coming?
which DB name are you using?

File: src/main/java/com/tek/trp/controller/CustomerController.java [hunk 2]
```diff
@@ -3,16 +3,15 @@
  */
 package com.tek.trp.controller;
```
Comment (laburli) [inline src/main/java/com/tek/trp/controller/CustomerController.java:4]: please send request and response for search customer to laburli@teksystems.com

**Review by raadari — APPROVED** [review]
approved

File: src/test/java/com/tek/trp/CustomerService/service/CustomerServices.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Before
+    public void setup(){
+        MockitoAnnotations.initMocks(this);
+    }
+
+
+    @Test
+    public void searchCustomerTestByIdAndName() throws CustomerNotFoundException {
```
Comment (laburli) [inline src/test/java/com/tek/trp/CustomerService/service/CustomerServices.java:45]: Nice Work...Meeta

**Review by laburli — APPROVED** [review]
Good work Meeta..keep it up:)

**Pull request merged**